build.gradle

```diff
@@ -270,7 +270,7 @@ dependencies {
   implementation group: 'net.javacrumbs.shedlock', name: 'shedlock-provider-jdbc', version: '4.12.0'
 
   implementation group: 'com.microsoft.azure', name: 'azure-storage', version: '8.6.5'
-  implementation group: 'com.microsoft.azure', name: 'azure-servicebus', version: '1.2.18', {
+  implementation group: 'com.microsoft.azure', name: 'azure-servicebus', version: '3.4.0', {
     exclude group: 'javax.mail', module: 'mail'
     exclude group: 'net.minidev', module: 'json-smart'
```

Comment:
should we review these exclusions? it well might be still needs to be here

Reply:
they are still there and still same versions, v1

Reply:
|         |    |    |    \--- net.minidev:json-smart:[1.3.1,2.3] -> 1.3.1|         
|    |    \--- com.sun.mail:javax.mail:1.6.1